Tests/KlaviyoSwiftTests/__Snapshots__/EncodableTests/testEventPayload.1.json

```diff
@@ -2,14 +2,30 @@
   "data" : {
     "attributes" : {
       "metric" : {
-        "name" : "blob"
+        "data" : {
+          "attributes" : {
+            "name" : "blob"
+          },
+          "type" : "metric"
+        }
       },
       "profile" : {
-        "email" : "blob@email.com",
-        "location" : {
-          "city" : "blob city"
-        },
-        "stuff" : 2
+        "data" : {
+          "attributes" : {
+            "anonymous_id" : "anon-id",
+            "email" : "blob@email.com",
+            "properties" : {
+
+            }
+          },
+          "meta" : {
+            "identifiers" : {
```

Comment:
meta identifiers is deprecated no longer needed.  You only have to pas the top part now (attributes with identifiers in them, like you have above)

Reply:
Oh weird I was just testing against the 7-15 alpha events API and it seemed to be necessary, will look again. 

(also damn that was a quick response, are you omniscient?)

Reply:
Ah, the change was made to /client/profiles but perhaps /client/events/ was not updated to match

I will investigate and see about removing it from there too.  In the meantime if it seems required, go ahead and leave it in here

Reply:
Oh, they are removing it but it is still open PR: https://github.com/klaviyo/app/pull/51575

The release is the same as yours though (7-15)

Reply:
Awesome, I'm sure that means we shouldn't have our new push tokens API use meta identifiers either then. So glad you were watching this repo thanks!